.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
For my understanding, can you please briefly tell what this would do. Thanks!

Reply:
Please correct me if I am wrong, does it mean that for all the file formats apply the below settings?

Reply:
root means that we won't continue up the path looking for more .editorconfg files and [*] means that these rules apply to all files.

Reply:
Thanks @sslaws